commands/curation/curationaudit.go

```diff
@@ -210,6 +244,23 @@ func (ca *CurationAuditCommand) doCurateAudit(results map[string][]*PackageStatu
 	return nil
 }
 
+func (ca *CurationAuditCommand) getRtManagerAndAuth(tech coreutils.Technology) (rtManager artifactory.ArtifactoryServicesManager, serverDetails *config.ServerDetails, err error) {
+	if ca.PackageManagerConfig == nil {
+		if err = ca.SetRepo(tech); err != nil {
+			return
+		}
+	}
+	serverDetails, err = ca.PackageManagerConfig.ServerDetails()
+	if err != nil {
+		return
+	}
+	rtManager, err = rtUtils.CreateServiceManager(serverDetails, 2, 0, false)
```

Comment:
Refactor values as consts.

Reply:
This is not a matter of consts in my opinion, thats what the function want, just read its signature